dkpro-core-stanfordnlp-gpl/src/test/java/de/tudarmstadt/ukp/dkpro/core/stanfordnlp/StanfordNamedEntityRecognizerTrainerTest.java

```diff
@@ -0,0 +1,133 @@
+        for (File file : trainingFiles) {
+            System.out.println("Training file: " + file.getAbsolutePath());
+        }
+
+        CollectionReaderDescription trainReader = createReaderDescription(Conll2002Reader.class,
+                Conll2002Reader.PARAM_PATTERNS, split.getTrainingFiles(),
+                Conll2002Reader.PARAM_LANGUAGE, ds.getLanguage(),
+                Conll2002Reader.PARAM_COLUMN_SEPARATOR,
+                Conll2002Reader.ColumnSeparators.TAB.getName(),
+                Conll2002Reader.PARAM_HAS_TOKEN_NUMBER, true, Conll2002Reader.PARAM_HAS_HEADER,
+                true, Conll2002Reader.PARAM_HAS_EMBEDDED_NAMED_ENTITY, true);
+
+        AnalysisEngineDescription trainer = createEngineDescription(
+                StanfordNamedEntityRecognizerTrainer.class,
+                StanfordNamedEntityRecognizerTrainer.PARAM_TARGET_LOCATION, model,
+                StanfordNamedEntityRecognizerTrainer.PARAM_PROPERTIES_LOCATION, properties,
+                StanfordNamedEntityRecognizerTrainer.PARAM_LABEL_SET, "noprefix",
+                StanfordNamedEntityRecognizerTrainer.PARAM_RETAIN_CLASS, true);
+
+        SimplePipeline.runPipeline(trainReader, trainer);
+
+        // Apply model and collect labels
+        System.out.println("Applying model to test data");
+        CollectionReaderDescription testReader = createReaderDescription(Conll2002Reader.class,
+                Conll2002Reader.PARAM_PATTERNS, split.getTestFiles(),
+                Conll2002Reader.PARAM_LANGUAGE, "de", Conll2002Reader.PARAM_COLUMN_SEPARATOR,
+                Conll2002Reader.ColumnSeparators.TAB.getName(),
+                Conll2002Reader.PARAM_HAS_TOKEN_NUMBER, true, Conll2002Reader.PARAM_HAS_HEADER,
+                true, Conll2002Reader.PARAM_HAS_EMBEDDED_NAMED_ENTITY, true,
+                Conll2002Reader.PARAM_READ_NAMED_ENTITY, true);
+
+        AnalysisEngineDescription ner = createEngineDescription(StanfordNamedEntityRecognizer.class,
+                StanfordNamedEntityRecognizer.PARAM_PRINT_TAGSET, true,
+                StanfordNamedEntityRecognizer.PARAM_MODEL_LOCATION, model);
+
+        List<Span<String>> actual = EvalUtil.loadSamples(iteratePipeline(testReader, ner),
+                NamedEntity.class, ne -> {
+                    return ne.getValue();
+                });
+        System.out.printf("Actual samples: %d%n", actual.size());
+
+        // Read reference data collect labels
+        ConfigurationParameterFactory.setParameter(testReader,
+                Conll2002Reader.PARAM_READ_NAMED_ENTITY, true);
+        List<Span<String>> expected = EvalUtil.loadSamples(testReader, NamedEntity.class, ne -> {
+            return ne.getValue();
+        });
+        System.out.printf("Expected samples: %d%n", expected.size());
+
+        Result results = EvalUtil.dumpResults(targetFolder, expected, actual);
+
+        // TODO how to determine the expected results beforehand?
```

Comment:
Ok thanks for explaining. I did exactly so, but I suspect the values will change now that I removed the test variable that restricted the training file to 30 sentences. I will have to see if it even runs on my machine now with the whole training file (will need a lot of memory), and if it does, I will adjust the numbers accordingly.